viewer/arvviewer.c

```diff
@@ -749,17 +767,66 @@ set_camera_widgets(ArvViewer *viewer)
 	string = g_strdup_printf ("%g", arv_camera_get_frame_rate (viewer->camera, NULL));
 	gtk_entry_set_text (GTK_ENTRY (viewer->frame_rate_entry), string);
 
+	// enables gtk widget based on feature representation
```

Comment:
Please remove the comment, the code is self explanatory

Reply:
updated in 1fd760c